packages/management/tanzu-auth/bundle/.imgpkg__BACKUP/README.md

```diff
@@ -11,8 +11,7 @@ Ensure you have built and pushed the `tanzu-auth-controller-manager` container t
 to the image in a later step.
 
 ```bash
-# currently called config-controller, rename coming soon
-cd path/to/tanzu-framework/addons/pinniped/config-controller
+cd path/to/tanzu-framework/addons/pinniped/tanzu-auth-controller
```

Comment:
Deleting in https://github.com/vmware-tanzu/tanzu-framework/pull/2249 so up to you if you want to ignore this to avoid a conflict :)

Reply:
yeah, I will, thanks!